v1/migration/index.php

```diff
@@ -19,16 +19,25 @@
 	// WordPress versions allowed for migration.
 	'wordpress' => [
 		'min'   => '4.9.0',
-		'max'   => '5.5.3',
+		'max'   => '5.7.2',
 		'other' => [
 			'#^4\.9$#',
-			'#^5\.5\.4-(alpha|beta|rc)#i',
-			'#^5\.6-(alpha|beta|rc)#i',
+			'#^5\.8-(alpha|beta|rc)#i',
 		],
 	],
 	// ClassicPress build to use for migration.
 	'classicpress' => [
 		'build'   => $build_url,
 		'version' => $version,
 	],
+	'plugins' => [
+		'wp-config-file-editor/wp-config-file-editor.php',
+		'disable-wp-core-updates-advance/disable-wp-core-updates-advance.php',
+		'disable-wordpress-updates/disable-updates.php',
+		'wp-downgrade/wp-downgrade.php',
+		'wordfence/wordfence.php',
```

Comment:
I don't think Wordfence being active should block a migration. There is only one feature of it that doesn't work:

> Wordfence doesn’t know how to scan ClassicPress core files for changes yet.
> \- https://docs.classicpress.net/user-guides/installing-classicpress/, "Plugin Conflicts" section

Reply:
I've updated my PR and removed WordFence.